trees/RobustTree.py

```diff
@@ -35,9 +37,10 @@ class RobustTreeClassifier(ClassifierMixin, BaseEstimator):
         The classes seen at :meth:`fit`.
```

Comment:
Could you please update the attributes here? We don't need to include all attributes but the most important ones like b, w, p, cost, budget, etc.